src/lib/Input.svelte

```diff
@@ -45,9 +41,20 @@
         innerTextContainsLetters();
 
     const handleKeyDown = (event: KeyboardEvent) => {
-        if (!canSend || event.key !== "Enter" || event.shiftKey) return;
-        send();
-        event.preventDefault();
+        let pressedEnter: boolean = event.key === "Enter";
```

Comment:
FYI these can be declared `const` (since they aren't mutated). It's also not totally necessary to explicit type these as `booleans` since they will be inferred as such, but it's certainly not wrong!

Reply:
True. Will do that.